### PR TITLE
DDI 413 [Contextual Doc] What is Amplitude 

### PR DESCRIPTION
**Pull request by Mercy811**
# Amplitude Developer Docs PR

## Description

preview: https://pr-593.d19s7xzcva2mw3.amplifyapp.com/analytics/what-is-amplitude/

In a side by side comparison with [Mixpanel](https://developer.mixpanel.com/docs/what-is-mixpanel) and [Segment](https://segment.com/docs/getting-started/), we lack conciseness on exactly what Amplitude is. 

- a paragraph of a high level context on what Amplitude is
- an Amplitude introduction video 
- a link to Amplitude Academy for new users as a good starting point 
- key concepts necessary to get started with Amplitude and links Amplitude Help Center

To be more concise and provider a high overview, TL;DR after the concept table for users not interested in the usage/implementation, for example, how amplitude track unique users. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/analytics/what-is-amplitude.md [hunk 1]
```diff
@@ -2,145 +2,54 @@
...
-Using Amplitude, you can track all the actions your users make in detail and better understand what’s working and what’s not.
+## Key concepts
 
-### What to track
+Amplitude is an event-based analytics tool that tracks the behaviors of users based on in-product interactions and analyzes user behavior in real-time. Event-based analytics is the method of tracking and analyzing interactions between users and your product, also known as events. Before you get started with Amplitude, it's important to know the key concepts used in Amplitude. 
 
-Amplitude gives you the power to decide what actions you want to track. You can choose to track anything and everything.
+|Name|Description|A music player app example|
+|---|-----------|-----------|
+|Users|A user is the specific individual that interacts with your product|A user uses the app to play music|
```
Comment (haoliu-amp) [inline docs/analytics/what-is-amplitude.md:17]: I think the order should be 
Events, Users, Event Props, User Props
OR
Events, Event Props, Users, User Props

**Review by haoliu-amp — CHANGES_REQUESTED** [review]

File: docs/analytics/what-is-amplitude.md [hunk 2]
```diff
@@ -2,145 +2,54 @@
...
+|Events|An event is an action a user has taken|A user presses the "Play Song" button|
+|User Properties|User properties are details about users|Whether a user is on a paid or free plan|
+|Event Properties|Event properties are details about events|The genre of the music|
+|Sessions|A session is the period of time a user has your app|A user uses the app to play music|
 
-For example, in Tunes, you could track the music control buttons the users presses or the number of songs each user has listened to in their sessions.
+TL;DR: If you aren't interested in the details of these concepts you can stop reading now. 
 
-#### Events represent actions users have taken
+### Users
```
Comment (haoliu-amp) [inline docs/analytics/what-is-amplitude.md:25]: Events section go first.

**Review by haoliu-amp — CHANGES_REQUESTED** [review]

File: docs/analytics/what-is-amplitude.md (hunk 1 above)
Comment (Mercy811) [reply docs/analytics/what-is-amplitude.md:17]: It makes sense to put events at the first place as it is the key of event-base analysis and also put event properties close to events. 

Changed it to Events, Event Props, Users, User Props

File: docs/analytics/what-is-amplitude.md (hunk 2 above)
Comment (Mercy811) [reply docs/analytics/what-is-amplitude.md:25]: Put Events section before Users. Also nested event properties under events and user properties under users to make closer to each other for better loop up and readability. 

Current structure: 
- Events
   - Event properties 
- Users 
   - User properties 
- Sessions

**Review by haoliu-amp — APPROVED** [review]

**Pull request merged**